status-im/status-react
# Add ability to Share image from Status

fixes #12472 

### Summary

This PR adds the ability to share an image to other apps from the image preview in Status.

#### Screenshots

<img width="446" src="https://user-images.githubusercontent.com/18485527/131609323-516bc8a0-427f-482d-b24e-dfc4f881aa3e.png">

<img width="446" src="https://user-images.githubusercontent.com/18485527/131609446-e66ab9df-e736-4a1c-b74b-89e367702be6.png">

#### Platforms

- Android
- iOS

##### Functional

- 1-1 chats
- public chats
- group chats

### Steps to test

- Open Status
- Send image from user A to user B
- Open image preview
- Verify share functionality and UI changes

status: ready
